### ISSUE TITLE
[Feature Request] - Investigate using relative links

### ISSUE THREAD

**Issue by nthState** [nthState/Haptrix#69]
# Investigate using relative links

We store absolute paths in AHAP files so that we can easily open any audio files.

Benefits
- [ ] Direct access to the file
- [ ] If you move the AHAP the link still works
- [ ] Sandboxing works

Drawbacks
- [ ] People with a shared workload may have different folder structures

Can we support relative links? ideally both?

**Comment by nthState** [comment]
https://developer.apple.com/documentation/corehaptics/representing_haptic_patterns_in_ahap_files
```
EventWaveformPath
The path string for the custom audio file, relative to the top of your bundle, such as MyAudioContent/Explosion.caf or its direct filename.
```

**Comment by nthState** [comment]
Released in 3.4
